README.md

```diff
@@ -4,14 +4,35 @@ This plugin helps to reformat existing codebase to adopted CodeStyle and work in
 
 ## Configuration steps
 
-1. Add this to your main ` build.gradle ` file : This should fail build operation if there exists codestyles adherence error(s).
+1. Adapt your ` build.gradle ` file to the following structure : This should fail build operation if there exists codestyles adherence error(s).
 ```
-apply plugin: 'checkstyle'
+   allprojects {
+    apply plugin: 'checkstyle'
 
     checkstyle {
-        config project.resources.text.fromUri('https://raw.githubusercontent.com/oyewaleoyelami/check/master/checkstyle.xml')
-        toolVersion = "8.11"
-    }
+               config project.resources.text.fromUri('https://raw.githubusercontent.com/oyewaleoyelami/check/master/checkstyle.xml')
```

Comment:
since repo is now public, you can grab xml from master

Reply:
fixed